BookStore.h

```diff
@@ -0,0 +1,135 @@
+#ifndef UNTITLED_BOOKSTORE_H
+#define UNTITLED_BOOKSTORE_H
+#ifndef UNTITLED_H
+#define UNTITLED_H
+
+#include "string"
+#include "vector"
+#include "algorithm"
+#include "iostream"
+
+class Book {
+public:
+    Book(std::string name,
+         int price_,
+         int numberOfPages_,
+         std::string author_,
+         int quantity_,
+         int numberOfSales_);
+    ~Book();
+
+    std::string getName(){
+        return name;
+    }
+    int getPrice(){
+        return price;
+    }
+    int getNumberOfSales(){
+        return numberOfSales;
+    }
+    std::string getAuthor(){
+        return author;
+    }
+
+private:
+    std::string name;
+    int price;
+    int numberOfPages;
+    std::string author;
+    int quantity;
+    int numberOfSales;
```

Comment:
member variables should be declared at the top of the class, followed by the member functions. This makes the code more organized and readable

Reply:
corrected